viadot/sources/databricks.py

```diff
@@ -0,0 +1,350 @@
+import json
+from pandas import DataFrame
+from viadot.config import local_config
+from viadot.sources.base import Source
+from delta.tables import *
+import pandas as pd
+import pyspark
+from pyspark import SparkConf, SparkContext
+import time
+# from viadot.sources import Base
+class Databricks(Source):
```

Comment:
Use Black to format the code; there should be two spaces before a class definition.